### PR TITLE
Fixes for RabbitMQ on SLE12

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: chef/cookbooks/rabbitmq/recipes/default.rb [hunk 1]
```diff
@@ -53,16 +53,17 @@
...
 bash "enabling rabbit management" do
   environment "HOME" => "/root/"
-  code "#{rabbitmq_plugins} enable rabbitmq_management > /dev/null"
+  code "#{rabbitmq_plugins} --offline enable rabbitmq_management > /dev/null"
   not_if "#{rabbitmq_plugins} list -E | grep rabbitmq_management -q", :environment => {"HOME" => "/root/"}
   notifies :restart, "service[rabbitmq-server]"
 end
 
+service "rabbitmq-server" do
+  supports :restart => true, :start => true, :stop => true, :status => true
```
Comment (vuntz) [inline chef/cookbooks/rabbitmq/recipes/default.rb]: Isn't that the default?

Comment (rsalevsky) [inline chef/cookbooks/rabbitmq/recipes/default.rb]: Chef Docs says:
>Default value: { :restart => false, :reload => false, :status => false } for all platforms (except for the Red Hat platform family, which defaults to { :restart => false, :reload => false, :status => true }.)

**Comment by rsalevsky** [comment]
+1

File: chef/cookbooks/rabbitmq/recipes/default.rb (hunk 1 above)
Comment (dirkmueller) [inline chef/cookbooks/rabbitmq/recipes/default.rb]: yeah, we're using the systemd provider (which seems to have status enabled by default) and on sle11 we use the red Hat family. so this change is a noop for SUSE. I'd still like to keep it though.

**Comment by vuntz** [comment]
+1

**Pull request merged**